.nvmrc

```diff
@@ -0,0 +1 @@
+v14.17.0
```

Comment:
Let's pin v12 for now, since that's what's in use on the Altis build environment.

Reply:
813d3c3 changes this to use node 12, and it _seems_ to work on my machine, but `npm install` throws some errors that seem to be about node-gyp, so I'm not sure how it'll behave in other environments. [node 12 is technically unsupported](https://github.com/dlmanning/gulp-sass#support) by `gulp-sass` so if this fails in a different env we may need to reconsider how to set up this build.

Is updating Altis to 14 (or 16 which iirc is the next LTS) on the roadmap?